contrib/spark/spark-operator/base/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - resources.yaml
+  - aggregated-roles.yaml
+namespace: kubeflow
+patches:
+  # Add securityContext to Spark Operator Pod.
+  - target:
+      kind: Deployment
+      labelSelector: "app.kubernetes.io/name=spark-operator"
+    patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/securityContext
+        value:
+          runAsUser: 1000
```

Comment:
The operator is based on spark image that uses 185 as non root user

Reply:
Good catch. Let me then remove this as we have the non-root user by default on Spark.

Reply:
Just runAsNonRoot:true is enough then